addons/kube-state-metrics/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+name: kube-state-metrics
+version: v0.1.0
+description: A simple service that listens to the Kubernetes API server and generates metrics about the state of the objects.
+icon: https://artifacthub.io/image/81797fec-309d-4380-a527-68f1c9e1dcfb@1x
+url: https://github.com/kubernetes/kube-state-metrics
+
+tags:
+ - Observability
+ - Kubernetes
+ - Metrics
+ - Prometheus
+
+invisible: false
+
+system:
+  vela: ">=v1.5.0"
```

Comment:
no body can install if requere the un-released version

Reply:
edited to >=v1.5.0-beta.4